migrations/versions/490ca3791aac_.py

```diff
@@ -0,0 +1,66 @@
+"""empty message
+
+Revision ID: 490ca3791aac
+Revises: 
+Create Date: 2021-03-17 13:38:33.981641
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '490ca3791aac'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('cache',
+    sa.Column('key', sa.String(length=100), nullable=False),
+    sa.Column('value', sa.PickleType(), nullable=True),
+    sa.Column('last_updated', sa.DateTime(), nullable=True),
+    sa.PrimaryKeyConstraint('key')
+    )
+    op.create_table('site_configuration',
+    sa.Column('key', sa.String(length=100), nullable=False),
+    sa.Column('value', sa.PickleType(), nullable=True),
+    sa.PrimaryKeyConstraint('key')
+    )
+    op.create_table('user',
+    sa.Column('id', sa.Integer(), autoincrement=False, nullable=False),
+    sa.Column('name', sa.String(length=500), nullable=True),
+    sa.Column('avatar_url', sa.String(length=500), nullable=True),
+    sa.Column('faculty', sa.Boolean(), nullable=True),
+    sa.Column('anonymous_by_default', sa.Boolean(), nullable=True),
+    sa.Column('autosave_timeout', sa.Integer(), nullable=True),
+    sa.Column('autosave_enabled', sa.Boolean(), nullable=True),
+    sa.Column('random_seed', sa.LargeBinary(length=32), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('nicety',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('end_date', sa.Date(), nullable=True),
+    sa.Column('author_id', sa.Integer(), nullable=True),
+    sa.Column('target_id', sa.Integer(), nullable=True),
+    sa.Column('anonymous', sa.Boolean(), nullable=True),
+    sa.Column('faculty_reviewed', sa.Boolean(), nullable=True),
+    sa.Column('starred', sa.Boolean(), nullable=True),
+    sa.Column('text', sa.Text(), nullable=True),
+    sa.Column('no_read', sa.Boolean(), nullable=True),
+    sa.Column('date_updated', sa.Text(), nullable=True),
+    sa.ForeignKeyConstraint(['author_id'], ['user.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('nicety')
+    op.drop_table('user')
+    op.drop_table('site_configuration')
+    op.drop_table('cache')
+    # ### end Alembic commands ###
```

Comment:
I worry a little bit about this; I don't believe in down migrations in general, and for this initial migration, allowing a down migration feels dangerous. Please feel free to push back on this if you disagree, but how would you feel about replacing these lines with a `pass`?

Reply:
Have done, and the other changes you requested in this file.